zoleotracker/databaseSQL.py

```diff
@@ -0,0 +1,57 @@
+import mysql.connector
+from mysql.connector import Error
+
+def create_db_connection(host_name, user_name, user_password, db_name):
+    connection = None
+    try:
+        connection = mysql.connector.connect(
+            host=host_name,
+            user=user_name,
+            passwd=user_password,
+            database=db_name
+        )
+        print("MySQL Database connection successful")
+    except Error as err:
+        print(f"Error: '{err}'")
+
+    return connection
+
+def check_for_table(connection, table_name):
```

Comment:
```suggestion
def table_exists(connection, table_name):
```
consider `exists` nomeclature